crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,183 @@
+in cases where mapping is not bidirectionally unambiguous.
+
+### `downloadUrl` <=> `repository-artifact`
+
+- CodeMeta `downloadUrl`: "If the file can be downloaded, URL to download the binary."
+- CFF `repository-artifact`: "The URL of the work in a build artifact/binary repository."
+
+While the scope of `repository-artifact` is documented to be somewhat narrower than `downloadUrl`, 
+its main use case is discovery (e.g. for reuse and reproducibility),
+which is arguably the same as for `downloadUrl`.
+Therefore the two can be assumed to signify the same.
+
+Future versions of CFF may document this more clearly (https://github.com/citation-file-format/citation-file-format/issues/326).
+
+### `softwareRequirements`/`softwareSuggestions` <=> `references`
+
+- CodeMeta `softwareRequirements`: "Required software dependencies"
+- CodeMeta `softwareSuggestions`: "Optional dependencies , e.g. for optional features, code development, etc"
+- CFF `references`: "Reference(s) to other creative works."
+
+#### CodeMeta to CFF
+
+The scope of `references` could be documented more clearly in CFF (https://github.com/citation-file-format/citation-file-format/issues/327).
+CFF's `references` is meant to work like a references list for a paper, 
+e.g., should include prior work that the work builds on.
+Required dependencies are interpreted to clearly constitute prior work that software builds on.
+Therefore, everything that is in `softwareRequirements` should go into `references`.
+
+Optional dependencies and dependencies needed for code development could be assumed 
+to build on the software (i.e., require prior existence of the software).
+Therefore, the software would constitute a `reference` *from* the `softwareSuggestion`,
+but not the other way round.
+I.e., `softwareSuggestions` should *not* be included in `references`.
+
+#### CFF to CodeMeta
+
+Not all CFF `references` can be assumed to be `softwareRequirements`,
+as `references` can contain arbitrary other work that the software builds on.
+The same is even more clearly true for `softwareSuggestions`.
+`references` therefore should not be included automatically in either, 
+without further manual verification.
+
+### `author` <=> `authors`
+
+CFF `authors` is an array of CFF `person` or CFF `entity` objects.
+
+### `citation` <=> `preferred-citation`/`references`/CFF root document
+
+- CodeMeta `citation`: "A citation or reference to another creative work, such as another publication, web page, scholarly article, etc."
+- CFF `preferred-citation`: "A reference to another work that should be cited instead of the software or dataset itself."
+- CFF `references`: "Reference(s) to other creative works."
+- CFF root document: The citation metadata for the software (or dataset) that the `CITATION.cff` file pertains to
+
+#### CodeMeta to CFF
+
+Understanding `citation` in CodeMeta to be the citation for the described software, the relationships are as follows:
```

Comment:
Is that correct though? From the definition it seems it must point to something else than the software itself.

Reply:
You're right. I've fixed this partly in an older commit for the CSV, and in the documentation in a007427e74f41c15d327886c1ff4a0b2714ddc10.